From e3064098291eff2dd6e429e2d4de955ca77c16e4 Mon Sep 17 00:00:00 2001
From: Ricardo Cerqueira <cyanogenmod@cerqueira.org>
Date: Sun, 19 Jan 2014 19:08:17 +0000
Subject: [PATCH 12/74] Break bootclasspath dependency

Use the library directly instead of forcing its global inclusion

Change-Id: I547c85b1b6c4ce09333e2eac2059cca1b851a346
---
 fmapp2/AndroidManifest.xml | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/fmapp2/AndroidManifest.xml b/fmapp2/AndroidManifest.xml
index 482a85e..3e9f7af 100644
--- a/fmapp2/AndroidManifest.xml
+++ b/fmapp2/AndroidManifest.xml
@@ -51,6 +51,10 @@
         android:allowTaskReparenting="true"
         android:taskAffinity="com.caf.task.fmradio"
         android:name=".FMAdapterApp">
+
+        <uses-library android:name="qcom.fmradio"
+            android:required="true" />
+
         <activity android:icon="@drawable/ic_launcher_fmradio"
             android:name=".FMRadio"
             android:label="@string/app_name"
-- 
2.17.1

